tools/interactive/interactivetool_wallace.xml

```diff
@@ -24,15 +24,14 @@
         </environment_variable> <!-- FIXME: We should have a better way to get user's API key -->
     </environment_variables>
     <command><![CDATA[
-        ln -s '${infile}' /srv/shiny-server/data/inputdata.txt &&
         /usr/bin/shiny-server.sh
     ]]>
     </command>
     <inputs>
-        <param name="infile" type="data" format="tabular,txt" label="Tabular in Clarion format"/>
+        <param name="infile" type="data" format="csv" label="csv file with for first columns: 'name', 'longitude', 'latitude' informations"/>
```

Comment:
but then also remove this input and the output

Reply:
yes, I'm on it! ;)